PhysicsTools/PythonAnalysis/python/rootplot/argparse.py

```diff
@@ -1,2353 +1,2352 @@
-# -*- coding: utf-8 -*-

```

Comment:
Shouldn't we be able to drop this file entirely by now ?

Reply:
I commented on that in my PR (well, for the RecoLuminosity one)..